torch/_subclasses/meta_utils.py

```diff
@@ -818,6 +824,7 @@ def _empty_create_subclass(
                             )
                             assert t.data is not None
                             _safe_copy(r.real_tensor, t.data)
+                    r.nested_int = t.nested_int
```

Comment:
it's bad that this propagation logic is in two places. Ideally I think `_empty_create_subclass()` would operate more recursively..